CHANGELOG.md

```diff
@@ -3,8 +3,17 @@ Changelog
 
 ##4.2
 
+Release date: Coming soon
+
 * Update Jetty from `9.4.5.v20170502` to `9.4.8.v20171121` [CHANGELOG](https://github.com/eclipse/jetty.project/releases/tag/jetty-9.4.8.v20171121)
-* User session memory leak. [JENKINS-49596](https://issues.jenkins-ci.org/browse/JENKINS-49596)
+
+## 4.1.1
+
+Release date: Feb 22, 2017
+
+* [JENKINS-49596](https://issues.jenkins-ci.org/browse/JENKINS-49596) -
+Prevent User session memory leak by setting the default idle session eviction timeout to 3 minutes. 
```

Comment:
Just noticed the default value, does this mean users get logged out of Jenkins if they are inactive for 3 minutes? That seems a little aggressive, from some googling it looks like 20-30 minutes is the usual default for other web servers. (The original reporter of the issue also set it to 30 minutes.)

Reply:
I am fine with changing default and spinning 4.1.2. @olamy WDYT?

Reply:
OK, will update